lenient/include/module.h

```diff
@@ -79,6 +81,8 @@ module_unload_context(void)            \
     this_private = NULL;               \
 }
 
+// TODO document the return values and consider returning true, when context
```

Comment:
Please add issue number of https://github.com/pottumuusi/c-encapsulation/issues/19 to this TODO comment.

Reply:
Instead of handling via the issue, please implement this TODO in this pull request.

Reply:
Implemented in: 92900006f8b608c9290089433ed9d7f22c1c34c6